tests/test_client.py

```diff
@@ -79,7 +79,30 @@ def test_subset_success(self):
         client = LaunchableClient("base_url", "org_name", "wp_name", "token", mock_requests, mock_subprocess)
         client.test_session_id = 1
 
-        got = client.subset(["tests/test1.py", "tests/test2.py"], "10")
+        got = client.subset(["tests/test1.py", "tests/test2.py"], None, "10")
+
+        expected_command = ['launchable', 'subset', '--session', '/test_sessions/1', '--target', '10%', 'file']
+        expected_input = 'tests/test1.py\ntests/test2.py'
+
+        mock_subprocess.run.assert_called_once_with(expected_command, input=expected_input, encoding='utf-8', stdout='PIPE', stderr='PIPE')
+        self.assertEqual(['tests/test2.py', 'tests/test1.py'], got)
+
+    def test_subset_success_with_options(self):
+        mock_output = MagicMock(name="output")
+        mock_subprocess = MagicMock(name="subprecess")
+
+        mock_subprocess.run.return_value = mock_output
+        mock_subprocess.PIPE = "PIPE"
+        # Success
+        mock_output.returncode = 0
+        mock_output.stdout = "tests/test2.py\ntests/test1.py\n"
+
+        mock_requests = MagicMock(name="requests")
```

Comment:
I recommend use [responses](https://github.com/getsentry/responses) to mock `requests`. If you refactor this test, consider it.

Reply:
Thanks! I'll definitely consider refactoring the tests using the library 👍